### PR TITLE
Fix master testing and update eslint-config

### PR DESCRIPTION
**Pull request by skjnldsv**
After #17263

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
:see_no_evil:

**Review by nickvergessen — APPROVED** [review]

**Comment by skjnldsv** [comment]
I never seen a pr so fast approved! :scream: :rocket:

**Comment by skjnldsv** [comment]
Weird

```
Changes not staged for commit:
4728 | (use "git add <file>..." to update what will be committed)
4729 | (use "git checkout -- <file>..." to discard changes in working directory)
4730 |  
4731 | modified: apps/workflowengine/js/workflowengine.js.map
4732

```

**Comment by skjnldsv** [comment]
Ah, now I get it :)
We should split the tests between two: compiling and git diff

**Comment by skjnldsv** [comment]
@nextcloud/javascript I need help here
https://drone.nextcloud.com/nextcloud/server/22087/2/3

I don't understand!

`modified: apps/workflowengine/js/workflowengine.js.map`

**Comment by ChristophWurst** [comment]
hold my php

**Comment by ChristophWurst** [comment]
should be fine now

**Comment by skjnldsv** [comment]
> should be fine now

any explanations?

**Comment by ChristophWurst** [comment]
> any explanations?

I ran

`npm i`

and then

`npm run build`

and the file in question was updated. So I amended.

**Comment by skjnldsv** [comment]
> and the file in question was updated. So I amended.

yes, I did that 10 times!
Every time the file change. Also, I used the compile bot just before, so it should have done the exact same thing than you did :)

**Comment by ChristophWurst** [comment]
Then let's see who's version wins :popcorn:

**Comment by skjnldsv** [comment]
> Then let's see who's version wins

Ahahahaha
https://drone.nextcloud.com/nextcloud/server/22089/2/3

told you :wink:

**Comment by skjnldsv** [comment]
Capture d’écran_2019-10-03_10-44-38

:rocket: :rocket: :rocket: :rocket: :rocket: :rocket: :rocket: :rocket: :rocket:

File: apps/workflowengine/templates/settings.php [hunk 1]
```diff
@@ -18,8 +18,10 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
+use OCA\WorkflowEngine\AppInfo\Application;
+style(Application::APP_ID, 'multiselect');
```
Comment (rullzer) [inline apps/workflowengine/templates/settings.php]: So... I'm happy it works now. But it still feels hacky. Including the CSS into the bundle is much nicer IMO (as it saves requests). But lets debug that later.

Comment (skjnldsv) [reply apps/workflowengine/templates/settings.php]: Let's ping @juliushaertl :see_no_evil:

**Comment by skjnldsv** [comment]
@danxuliu there are some acceptance tests failures.
No idea if they were here before or not. Could you check please?

**Comment by skjnldsv** [comment]
> @danxuliu there are some acceptance tests failures.
> No idea if they were here before or not. Could you check please?

SHould be fixed now :)

**Comment by skjnldsv** [comment]
Also fixed the acceptance-app-files-tags ^^

**Comment by ChristophWurst** [comment]
I would appreciate if we could have a second person review & approve the changes that happen after the reviews :)

**Comment by skjnldsv** [comment]
@ChristophWurst but there is 2 reviews :thinking:

File: apps/workflowengine/templates/settings.php (hunk 1 above)
Comment (juliushaertl) [reply apps/workflowengine/templates/settings.php]: Added it to the list to fix in https://github.com/nextcloud/server/issues/12790

**Comment by danxuliu** [comment]
>  @danxuliu there are some acceptance tests failures.
>  No idea if they were here before or not. Could you check please?

They came from #17263. And I missed the mention and this pull request; I found the acceptance tests failures in another one. So after manually bisecting the commit (please, next time remove the ending semicolon in one commit, add the missing spaces in other, fix the documentation parameters in other... ;-) ) I finally found that the issue was caused by the [destructuring assignment](https://github.com/nextcloud/server/pull/17263/files#diff-62d91b5a8aa522571387ddb425be596eR64). And when I was happily going to open the pull request to fix it... then I saw that [you already fixed it here](https://github.com/nextcloud/server/pull/17367/commits/972279d31b15dfe972eb5c254ed8b56b7d926d72#diff-62d91b5a8aa522571387ddb425be596eL64)! I want to cry :-P

Jokes aside, I would have expected Babel to transpile the destructuring assignment to something that older browsers could handle (as I guess that the same issue would happen in Internet Explorer 11). Maybe some rule or plugin is missing?

**Comment by skjnldsv** [comment]
> I want to cry :-P

/me pat @danxuliu in the back :grin: 

Yeah, it's a bit weird! I reverted back the fix of the detructuring assignment and ignoroed eslint warning as this is a weird hack for old firefox entries. We shouldn't mess with this too much :)

**Pull request merged**